tcf_website/models/models.py

```diff
@@ -300,6 +300,10 @@ def average_difficulty(self):
         return Review.objects.filter(course=self).aggregate(
             models.Avg('difficulty'))['difficulty__avg']
 
+    def review_count(self):
```

Comment:
I believe you could also do `self.review_set.count()`

Reply:
I wasn't gonna change it so the code stays consistent w/ the other course model methods

Reply:
Hmm I'd argue the other methods (that I wrote haha) should be refactored to use `self.review_set.X` as it seems more readable

Reply:
done